infra.bs

```diff
@@ -644,6 +646,35 @@ Standard that should be reported and addressed.
 </div>
 
 
+<h3 id=ecma-262-abstract-operations>ECMA-262 Abstract Operations</h3>
+
+<p>ECMA-262 uses a specification type named [=Completion Record=] to model exception-based control
```

Comment:
Completion records aren't *just* for modelling exceptions-based flow; even if ECMAScript didn't have exceptions, it would still need completion records (or something like them) to model break+continue+return control flow.